sphinx/developers/java-library.rst

```diff
@@ -5,7 +5,7 @@ Bio-Formats as a Maven, Gradle or Ivy dependency
 ------------------------------------------------
 
 All released :file:`.jar` artifacts are published to and can be obtained from
-the `OME Artifactory <http://artifacts.openmicroscopy.org/artifactory>`_.  The
+the `OME Artifactory <https://artifacts.openmicroscopy.org/artifactory>`_.  The
```

Comment:
Given the redirect, should this more simply be https://artifacts.openmicroscopy.org/?